abi-bindings/go/packer/packer_test.go

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package packer
+
+import (
+	"fmt"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"math/big"
+	"math/rand"
+	"reflect"
+	"testing"
+
+	validatorsetsig "github.com/ava-labs/teleporter/abi-bindings/go/governance/ValidatorSetSig"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/teleporter/TeleporterMessenger"
+	teleporterregistry "github.com/ava-labs/teleporter/abi-bindings/go/teleporter/registry/TeleporterRegistry"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/tools/go/packages"
+)
+
+const interfaceName = "ABIPacker"
+
+// Paths below are relative to the current package.
+const packagePath = "../..."
+const currentPackagePath = "."
+
+var fs = token.NewFileSet()
+
+var packerTypes = map[string]ABIPacker{
+	"ValidatorSetSigMessage": &validatorsetsig.ValidatorSetSigMessage{},
```

Comment:
If we ever have multiple tests in here that modify the values in this map, then the exhaustiveness test may incorrectly pass. For instance if an additional test runs before `TestExhaustivePacking` that populates a field that is erroneously left out of the corresponding `Pack` method, then `TestExhaustivePacking` would still pass.

Reply:
Cleared this up. The values are only used to get an instance of a concrete type for reflection.